Competition2018/src/main/java/competition/subsystems/offboard/OffboardFramePackingUtils.java

```diff
@@ -20,7 +22,19 @@
     
     public static byte[] packSetCommandFrame(int commandId) {
         return new byte[] {
-            (byte)(commandId & 0xFF)
+                (byte)(commandId & 0xFF)
         };
     }
-}
+    
+    public static byte[] packScoringPlacement(ArrayList<Integer> scoringPlacement) {
+    	int sequence = 0;
+    	for(int i = 0; i < 3; i++) {
+    		int scoring = (scoringPlacement.get(i) << (2-i));
```

Comment:
The `2-` part is a bit weird; I'd suggest changing the loop to be `int i = 3; i >= 1; i--`

Reply:
!